pkg/utils/utils_mlx.go

```diff
@@ -0,0 +1,116 @@
+package utils
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/golang/glog"
+)
+
+//BlueField2 mode representation
```

Comment:
the mode and the parameters will be valid for future Bluefield NICs. so maybe drop the "2"